palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/model/PalisadeClientRequest.java

```diff
@@ -33,10 +33,10 @@
  * Represents the original data that has been sent from the client to Palisade Service for a request to access data.
  * This data will be forwarded to a set of services with each contributing to the processing of this request.
  * This version represents the original request.
- * Next in the sequence is the input for user-service where this data will be used as a request for a User.
+ * Next in the sequence is the input for User-Service where this data will be used as a request for a User.
  * Note there are two classes that effectively represent the same data but represent a different stage of the process.
```

Comment:
This note is no longer correct with the refactoring.  It should now read that there are three classes and list 
uk.gov.gchq.palisade.service.palisade.model.PalisadeClientRequest
uk.gov.gchq.palisade.service.palisade.model.PalisadeSystemResponse
uk.gov.gchq.palisade.service.user.request.UserRequest.  This will also need to be checked in the User Service

Reply:
updated